opsdroid/connector/tests/test_connector_rocketchat.py

```diff
@@ -0,0 +1,295 @@
+"""Tests for the RocketChat connector."""
+import asyncio
+import pytest
+import contextlib
+import asynctest.mock as amock
+
+from opsdroid.core import OpsDroid
+from opsdroid.connector.rocketchat import RocketChat
+from opsdroid.events import Message
+from opsdroid.cli.start import configure_lang
+
+configure_lang({})
+
+
+def test_init():
+    """Test that the connector is initialised properly."""
+    connector = RocketChat(
+        {
+            "name": "rocket.chat",
+            "token": "test",
+            "user-id": "userID",
+            "update-interval": 0.1,
+        },
+        opsdroid=OpsDroid(),
+    )
+    assert connector.default_target == "general"
+    assert connector.name == "rocket.chat"
+
+
+def test_missing_token(caplog):
+    """Test that attempt to connect without info raises an error."""
+
+    RocketChat({})
+    assert "Unable to login: Access token is missing." in caplog.text
+
+
+def setUp():
+    configure_lang({})
+    connector = RocketChat(
+        {
+            "name": "rocket.chat",
+            "token": "test",
+            "user-id": "userID",
+            "default_target": "test",
+        },
+        opsdroid=OpsDroid(),
+    )
+
+    connector.latest_update = "2018-10-08T12:57:37.126Z"
+
+    with amock.patch("aiohttp.ClientSession") as mocked_session:
+        connector.session = mocked_session
+
+
+@pytest.mark.asyncio
+async def test_connect(capsys):
+    connector = RocketChat({})
```

Comment:
By passing an empty dict into `RocketChat` you are passing and empty config option to the connector. That's why it's complaining about not having a token.

We can pass the config from the `setUp` test

```suggestion
    connector = RocketChat(
    	{
    		"name": "rocket.chat",
    		"token": "test",
    		"user-id": "userID",
    		"default_target": "test
    	}
    )
```

_Note: I'm using the browser to review and used tabs on this suggestion - not sure if committing this suggestion won't send those tabs and break python._